openfisca_us/parameters/gov/states/in/tax/income/deductions/nonpublic_school/amount.yaml

```
@@ -0,0 +1,12 @@
+description: IN deduction for qualified dependent children in private, parochial, or home school with unreimbursed expense
+values:
+  2021-01-01: 1_000
+metadata:
+  period: year
+  unit: currency-USD
+  name: in_nonpublic_school_deduction
+  label: IN nonpublic school deduction
+  reference:
+    - title: IC 6-3-2-22 Deduction; unreimbursed education expenditures
+      href: http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3-2-4 (d)(1)
```

Updating and making a valid URL (it gets linked in https://policyengine.org/us/api-explorer)
```suggestion
      href: http://iga.in.gov/legislative/laws/2021/ic/titles/006#6-3-2-22
```